source/codegen/metadata/nixnetsocket/functions.py

```diff
@@ -173,6 +173,52 @@
         ],
         'returns': 'int32_t'
     },
+    'RecVFrom': {
```

Comment:
I think this should be "RecvFrom" like "Recv".

Reply:
whoops forgot to change it in this one as well. Fixed